File: docs/language_bootcamp/java/2_ControlFlow/Fizzbuzz.java
public class Fizzbuzz {
    
    public static void main(String[] args)
    {
        int n = 20;

        for(int i = 1; i <= n; i++) {

            System.out.print(i + " ");

            if(i % 3 == 0 && i % 5 == 0)
                System.out.print("Fizzbuzz!");
            else if(i % 3 == 0)
                System.out.print("Fizz");
            else if(i % 5 == 0)
                System.out.print("Buzz");

            System.out.println();

        }
    }

}

File: docs/language_bootcamp/python/src/FizzBuzz.py
#!/usr/bin/python3

def fizzBuzz(index: int) -> str:
    '''
    Reminder about what fizzbuzz is
    Write a program that prints the numbers from 1 to 100. But for multiples of
    three print “Fizz” instead of the number and for the multiples of five
    print “Buzz”. For numbers which are multiples of both three and five print
    “FizzBuzz”."
    '''
    out = str(index)
    if index % 15 == 0:
        out = "FizzBuzz"
    elif index % 3 == 0:
        out = "Fizz"
    elif index % 5 == 0:
        out = "Buzz"
    return out


for i in range(1, 100):
    print(fizzBuzz(i))
